server/src/main/java/org/opensearch/action/admin/cluster/shards/routing/weighted/get/ClusterGetWeightedRoutingResponse.java

```diff
@@ -31,26 +31,37 @@
  * @opensearch.internal
  */
 public class ClusterGetWeightedRoutingResponse extends ActionResponse implements ToXContentObject {
-    private WeightedRouting weightedRouting;
-    private String localNodeWeight;
-    private static final String NODE_WEIGHT = "node_weight";
+    public WeightedRouting getWeightedRouting() {
+        return weightedRouting;
+    }
+
+    private final WeightedRouting weightedRouting;
 
-    public String getLocalNodeWeight() {
-        return localNodeWeight;
+    public Boolean getDiscoveredMaster() {
```

Comment:
lets use `cluster_manager` everywhere . `master` is going to get deprecated in 3.0.